weave/trace_server/migrations/005_add_cost.up.sql

```diff
@@ -0,0 +1,49 @@
+CREATE TABLE llm_token_prices (
```

Comment:
how does this get populated? Should we have a mechanism of bootstrapping this / updating from some public location?